hazelcast/src/main/java/com/hazelcast/internal/management/request/ExecuteScriptRequest.java

```diff
@@ -124,7 +119,27 @@ public void fromJson(JsonObject json) {
         for (JsonValue target : getArray(json, "targets", new JsonArray())) {
             targets.add(target.asString());
         }
-        targetAllMembers = getBoolean(json, "targetAllMembers", false);
-        bindings = new HashMap<String, Object>();
+    }
+
+    private static void addSuccessResponse(JsonObject root, StringBuilder scriptResult,
+                                           Address address, String result) {
+
+        addResponse(root, scriptResult, address, true, result);
+    }
+
+    private static void addErrorResponse(JsonObject root, StringBuilder scriptResult,
+                                         Address address, Exception e) {
+        addResponse(root, scriptResult, address, false, ExceptionUtil.toString(e));
```

Comment:
Do we have to append all the exception results to the `scriptResult` field? Since the `scriptResult` is kept for compatibility and the goal is to avoid displaying exceptions on the UI, could we skip appending exceptions, so even older mancenter versions will not display stack traces? Another option could be to just append the exception message instead of the whole stack trace: this way we still indicate an error occurred and avoid showing stack traces. wdyt?

Reply:
>  the goal is to avoid displaying exceptions on the UI

I think we should still have the full stacktrace, otherwise we wouldn't be able to show the exception details in logs either (which would limit usability of the scripting tab). As this is a security fix, we can always ask the user to upgrade to latest to have full security.

I have addressed all other comments. Thanks for the review.